study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/TestBostonAddNoSymptionsOption.java

```diff
@@ -0,0 +1,105 @@
+import org.broadinstitute.ddp.util.ConfigUtil;
+import org.jdbi.v3.core.Handle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestBostonAddNoSymptionsOption implements CustomTask {
+
+    private static final Logger LOG = LoggerFactory.getLogger(BrainPrequalV2.class);
+    private static final String DATA_FILE = "patches/adhoc-no-symptom.conf";
+
+    private Path cfgPath;
+    private Config cfg;
+    private Config varsCfg;
+    private Config dataCfg;
+    @Override
+    public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
+        this.cfgPath = cfgPath;
+        this.cfg = studyCfg;
+        this.varsCfg = varsCfg;
+
+        File file = cfgPath.getParent().resolve(DATA_FILE).toFile();
+        if (!file.exists()) {
+            throw new DDPException("Data file is missing: " + file);
+        }
+        this.dataCfg = ConfigFactory.parseFile(file).resolveWith(varsCfg);
+    }
+
+    @Override
+    public void run(Handle handle) {
+        String studyGuid = "testboston";
+        String activityCode = "ADHOC_SYMPTOM";
+        StudyDto studyDto = handle.attach(JdbiUmbrellaStudy.class).findByStudyGuid("testboston");
+
+        ActivityDao activityDao = handle.attach(ActivityDao.class);
+
+        ActivityVersionDto currentActivityVerDto = activityDao.getJdbiActivityVersion().findByActivityCodeAndVersionTag(
+                studyDto.getId(), activityCode, "v1").get();
+
+
+        // find the adhoc symptom activity
+        ActivityDto activityDto = handle.attach(JdbiActivity.class)
+                .findActivityByStudyGuidAndCode(studyGuid, activityCode)
+                .orElseThrow(() -> new DDPException("Could not find id for activity " + activityCode + " and study id " + studyGuid));
+
+        ActivityDef currActivityDef = activityDao.findDefByDtoAndVersion(activityDto, currentActivityVerDto);
+        FormActivityDef currFormActivityDef = (FormActivityDef)currActivityDef;
+
+        PicklistQuestionDao plQuestionDao = handle.attach(PicklistQuestionDao.class);
+        JdbiQuestion jdbiQuestion = handle.attach(JdbiQuestion.class);
+
+        // find the symptoms question
+        QuestionDto symptomsQuestionDto = jdbiQuestion.findLatestDtoByStudyIdAndQuestionStableId(studyDto.getId(), "ADHOC_SYMPTOMS").get();
+
+        List<FormSectionDef> bodySections = currFormActivityDef.getSections();
+        FormSectionDef plQuestionSec = bodySections.get(0);
+        FormBlockDef formBlockDef = plQuestionSec.getBlocks().get(0);
+
+        // load the new option config
+        Config option = dataCfg.getConfig("option");
+
+        var newOption = new Gson().fromJson(ConfigUtil.toJson(option), PicklistOptionDef.class);
+
+        ActivityVersionDto versionDto = handle.attach(JdbiActivityVersion.class)
+                .getActiveVersion(currFormActivityDef.getActivityId())
+                .orElseThrow(() -> new DDPException("Could not find latest version for activity " + activityCode));
```

Comment:
There's only one version of AdHoc Symptom activity right now, so we can either use this or use the `currentActivityVerDto` that we have already found above.